sky/utils/common_utils.py

```diff
@@ -9,12 +9,23 @@
 import socket
 import sys
 import time
+import uuid
 import yaml
 
 from sky import sky_logging
 
 logger = sky_logging.init_logger(__name__)
 
+_run_id = None
+
+
+def get_run_id():
+    """Returns a unique run id for this logging."""
```

Comment:
Update doc?
- rm 'for this logging'
- perhaps expand on the definition: e.g., does the run id preserve across a spot job's recovery attempts?

Reply:
Updated the docs. For the unique id for across a spot job's recovery attempts, I will have another PR adding that ;)